internal/gatewayapi/backendtrafficpolicy.go

```diff
@@ -425,6 +425,9 @@ func (t *Translator) buildLoadBalancer(policy *egv1a1.BackendTrafficPolicy) *ir.
 		lb = &ir.LoadBalancer{
 			LeastRequest: &ir.LeastRequest{},
 		}
+		if policy.Spec.LoadBalancer.SlowStartWindow != nil {
```

Comment:
can you also add some test under https://github.com/envoyproxy/gateway/tree/main/internal/gatewayapi/testdata

Reply:
done